pkg/test/framework/components/namespace/kube.go

```diff
@@ -57,7 +57,7 @@ func (n *kubeNamespace) Dump(ctx resource.Context) {
 		return
 	}
 
-	kube2.DumpPods(n.ctx, d, n.name)
+	kube2.DumpPods(n.ctx, d, n.name, []string{"k8s-app=istio-cni-node"})
```

Comment:
doesn't this mean we will only match CNI? so we will lose all the namespace dumping (echo).I am thinking the istio/ ones are sufficient possibly?

Reply:
Ah yeah. Good catch. I meant to put empty selector here.